addons/stock/models/stock_move.py

```diff
@@ -295,6 +303,31 @@ def _compute_product_qty(self):
             move.product_qty = move.product_uom._compute_quantity(
                 move.product_uom_qty, move.product_id.uom_id, rounding_method='HALF-UP')
 
+    @api.depends('product_packaging_id', 'product_uom', 'product_uom_qty')
+    def _compute_product_packaging_qty(self):
+        for line in self:
+            if not line.product_packaging_id:
+                line.product_packaging_qty = False
+            else:
+                packaging_uom = line.product_packaging_id.product_uom_id
+                packaging_uom_qty = line.product_uom._compute_quantity(line.product_uom_qty, packaging_uom)
+                line.product_packaging_qty = float_round(
+                    packaging_uom_qty / line.product_packaging_id.qty,
+                    precision_rounding=packaging_uom.rounding)
+
+    @api.depends('product_packaging_id', 'product_uom', 'quantity_done')
+    def _compute_product_packaging_qty_done(self):
+        for line in self:
+            if not line.product_packaging_id:
+                line.product_packaging_qty_done = False
+            else:
+                packaging_uom = line.product_packaging_id.product_uom_id
+                packaging_uom_qty = line.product_uom._compute_quantity(line.quantity_done, packaging_uom)
+                line.product_packaging_qty_done = float_round(
+                    packaging_uom_qty / line.product_packaging_id.qty,
+                    precision_rounding=packaging_uom.rounding)
```

Comment:
To speed up the assignment you can change this to:

```
self.product_packaging_qty_done = False
for line in self:
    if not line.product_packaging_id:
        continue
    packaging_uom = line.product_packaging_id.product_uom_id
    packaging_uom_qty = line.product_uom._compute_quantity(line.quantity_done, packaging_uom)
    line.product_packaging_qty_done = float_round(
                packaging_uom_qty / line.product_packaging_id.qty,
                precision_rounding=packaging_uom.rounding)
```

This way the assignment of false to all the lines it doesn't apply to is done once. Same comment applies to `_compute_product_packaging_qty`

Reply:
changed as recommended in bdf7da8